index.js

```diff
@@ -1,95 +1,122 @@
 // ***** Question 1 *****
+var counter = 1;
```

Comment:
`var` works here, but get in the habit of using `let` instead (`var` has some tricky behavior, so you won't see it used as much in modern Javascript).